karabo_data/tests/make_examples.py

```diff
@@ -255,13 +255,16 @@ def make_spb_run(dir_path, raw=True):
                 [
                     XGM('SA1_XTD2_XGM/DOOCS/MAIN'),
                     XGM('SPB_XTD9_XGM/DOOCS/MAIN'),
-                ], ntrains=32, chunksize=32)
+                    BaslerCamera('SPB_IRU_CAM/CAM/SIDEMIC',
+                        sensor_size=sensor_size), ], ntrains=32, chunksize=32)
     write_file(osp.join(dir_path,
                         '{}-R0238-DA01-S00001.h5'.format(prefix)),
                 [
                     XGM('SA1_XTD2_XGM/DOOCS/MAIN'),
                     XGM('SPB_XTD9_XGM/DOOCS/MAIN'),
-                ], ntrains=32, firsttrain=10032, chunksize=32)
+                    BaslerCamera('SPB_IRU_CAM/CAM/SIDEMIC',
+                        sensor_size=sensor_size), ], ntrains=32,
```

Comment:
The position of the `]` here makes it hard to see where the list ends - can you tweak the formatting a bit?